ci-operator/step-registry/telcov10n/functional/compute-nrop/ocp-operators/telcov10n-functional-compute-nrop-ocp-operators-commands.sh

```
@@ -0,0 +1,50 @@
+        DEST_FILE=$( basename $file | cut -d'_' -f3 )
+        cp $file ${ECO_CI_CD_INVENTORY_PATH}/$DEST_DIR/$DEST_FILE
+    fi
+done
+
+echo "Set CLUSTER_NAME env var"
+if [[ -f "${SHARED_DIR}/cluster_name" ]]; then
+    CLUSTER_NAME=$(cat "${SHARED_DIR}/cluster_name")
+fi
+
+export CLUSTER_NAME=${CLUSTER_NAME}
+echo CLUSTER_NAME=${CLUSTER_NAME}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)